wrims/build.gradle

```diff
@@ -0,0 +1,56 @@
+plugins{
+    id 'library.deps-conventions'
+    id 'library.java-conventions'
+
+    // The following plugin is used to build the OSGI bundle
+    id 'biz.aQute.bnd.builder' version '6.4.0'
+}
+
+repositories {
+    mavenLocal()
+}
+
+java {
+    sourceCompatibility = JavaVersion.VERSION_1_8
+    targetCompatibility = JavaVersion.VERSION_1_8
+}
+
+sourceSets {
+    src {
+        java {
+            srcDirs "src/main/java/"
```

Comment:
Isn't this the default? Seems like this entire sourceSet block can be skipped

Reply:
removed